TechnicoBackend/Services/UserService.cs

```diff
@@ -0,0 +1,64 @@
+﻿using TechnicoBackend.Repositories;
+using TechnicoBackend.Models;
+
+namespace TechnicoBackend.Services
+{
+    public class UserService
```

Comment:
Consider using an interface IUserService from which the current class would be inherited.